.github/workflows/ci.yml

```diff
@@ -29,3 +29,19 @@ jobs:
         env:
           CGO_ENABLED: 0
         run: go test -v ./...
+
+  test-install:
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        shell: [sh, bash, zsh]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Print build information
+        run: "echo head_ref: ${{ github.head_ref }}, ref: ${{ github.ref }}"
+
+      - uses: actions/checkout@v3
+
+      - name: Test install.sh script
+        run: ${{ matrix.shell }} ./install.sh
```

Comment:
How do you know this works?

Reply:
Add some assertions like running the server post installation and possibly `tee` the output and grepping on expected strings.

Reply:
it's going to throw 127 error code if `temporal` is not found. Otherwise if `temporal -h` succeeds it ends with 0

Reply:
slightly updated the action